.github/workflows/build-push-gotrue.yml

```diff
@@ -0,0 +1,60 @@
+name: publish docker image - gotrue
+
+on:
+  pull_request:
+    paths:
+      - Dockerfile*
+      - .github/workflows/*.yml
+  push:
+    branches:
+      - main
+
+env:
+  QUAY_REPO: "quay.io/tigrisdata/gotrue"
+
+jobs:
+  build-image:
+    name: Build and Push image
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Get short github SHA
+        id: var
+        shell: bash
+        run: |
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
```

Comment:
This syntax is deprecated.

Reply:
Please review https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
We should do a cursory sweep to see if `set-output` is being used widely and fix it where possible. The switch to the new syntax is not bad. You can check in the build workflows I submitted recently to `go/terraform-modules` and `go/infra`.

Reply:
Thank you. Changed to env file